ui/raidboss/data/06-ew/raid/p8s.ts

```diff
@@ -75,9 +75,9 @@ export const positionTo8Dir = (combatant: PluginCombatantState) => {
   return Math.round(4 - 4 * Math.atan2(x, y) / Math.PI) % 8;
 };
 
-export const headingTo4Dir = (heading: number) => {
-  // N = 0, E = 1, etc
-  return Math.round(2 - 2 * heading / Math.PI) % 4;
+export const headingTo8Dir = (heading: number) => {
+  // Dirs: N = 0, N = 1, ..., W = 7
```

Comment:
If N = 0, should NE = 1?  And would NW = 7?  Or are these being done in pairs (e.g. N=[0,1], E=[2,3], etc)?

Reply:
oh it's a typo in the comment, NE is 1

Reply:
thanks

Reply:
Np.  I think you might still have a typo on W = 7 though.

Reply:
oh man, that's embarrassing.  I've fixed it to be NW.